example-02/CMakeLists.txt

```diff
@@ -0,0 +1,32 @@
+cmake_minimum_required(VERSION 3.17 FATAL_ERROR)
```

Comment:
Why is CMake 3.17 such a strong requirement?

Reply:
The

```cmake
find_package(CUDAToolkit)
```

used below simplifies the setting up of the project, but is only available on latest cmake.